Satellite-im/UplinkWeb
# chore(warning): fix warning

### What this PR does 📖

- Fix this warning

<img width="1492" alt="Captura de ecrã 2024-09-19, às 15 31 39" src="https://github.com/user-attachments/assets/28547871-1696-4cad-9306-fb2cec715e85">


how to test
- npm run build
- warning should not appear
- typing indicator should be the same
